### PR TITLE
Add Actual trial length to caseworker view

### PR DESCRIPTION
**Pull request by kmahern**
#### What
Fix bug resulting from CBO-471 where Actual Trial Length was no longer being displayed to caseworkers for litigator claims on the case summary screen

#### Ticket
CBO-542 LGFS actual trial length caseworker view (https://dsdmoj.atlassian.net/projects/CBO/issues/CBO-542)

#### Why
Caseworkers need to see this field for lgfs graduated fees in order to process claims.

#### How
This value was originally displayed in the Case Details section, I have moved it to the Fees section on the caseworker view in order to be more consistent with the litigator check your claim page and also reflect that the actual trial length is entered as part of the litigator graduated fee page.

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -1715,6 +1715,7 @@ en:
       distance: Distance
       fee_category: Fee category
       fee_type: Fee type
+      actual_trial_length: Actual trial length
```
Comment (jsugarman) [inline config/locales/en.yml]: you can share translation values using anchors, `&`

define in dictionary:

```
actual_trial_length: &actual_trial_length Actual trial length
```
and use further down as required
```
actual_trial_length: *actual_trial_length
```

**Review by jsugarman — APPROVED** [review]
👍 looks good

1 suggestion of sharing common translations as a general refactor/tidy (worth doing on things you are touching anyway)

Are there any feature tests for these views? to discuss

**Comment by kmahern** [comment]
@jsugarman Thanks for the review! I have made the changes to add actual trial length as a common translation (/config/locales/en.yml). I have had a look and I don't believe there are currently any feature tests for these views, do you think we are OK to go ahead without?

File: config/locales/en.yml [hunk 2]
```diff
@@ -96,6 +96,7 @@ en:
         middle_name: &middle_name 'Middle name'
         last_name: &last_name 'Last name'
       fees:
+        actual_trial_length: &actual_trial_length Actual trial length
```
Comment (jsugarman) [inline config/locales/en.yml:99]: ace 👍

File: app/views/shared/_summary_lgfs.html.haml [hunk 3]
```diff
@@ -16,7 +16,7 @@
               = t('.fee_category')
             %th{ scope: 'col' }
               = t('.fee_type')
-            - if claim.transfer? || (claim.final? && !claim.fixed_fee_case?) # could be applied to graduated too
+            - if claim.transfer? || (claim.final? && !claim.fixed_fee_case?)
```
Comment (jsugarman) [inline app/views/shared/_summary_lgfs.html.haml:19]: 👍 cool, as discussed you can merge and then do a separate PR to move this conditional logic into presenter(s) - employing either inheritance in fee presenters(s) or placing logic in `base_claim_presenter`

**Review by jsugarman — APPROVED** [review]
👍
